src/test/java/com/jcabi/s3/AwsBucketITCase.java

```diff
@@ -147,4 +150,17 @@ public void listsInPrefixedBucketWithouCollisions() throws Exception {
         }
     }
 
+    /**
+     * AwsBucket can check if the bucket exists.
+     * @throws IOException If fails
+     */
+    @Test
+    public void checkBucketExistence() throws IOException {
+        final Bucket bucket = this.rule.bucket();
+        Assert.assertTrue(bucket.exists());
```

Comment:
@yegor256 that's because I wanted to test a scenario when bucket doesn't exist, the simplest way to do it is to remove an existing bucket, but `BucketRule.drop()` is a `private` method, therefore I came to use `aws.deleteBucket()`